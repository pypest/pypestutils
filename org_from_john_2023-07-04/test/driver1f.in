32
ex-gwf-u1disv.cbc
2
driver1f.out


