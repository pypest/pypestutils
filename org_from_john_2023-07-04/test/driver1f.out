     KSTP      KPER              TEXT        NCPL      INT1      INT2     IMETH      DELT             PERTIM            TOTIM                   TXT1ID1             TXT2ID1             TXT1ID2             TXT2ID2
        1         1      FLOW-JA-FACE         601         1        -1         1    1.00000000        1.00000000        1.00000000    
        1         1        DATA-SPDIS         121         1        -1         6    1.00000000        1.00000000        1.00000000         EX-GWF-U1DISV                 NPF       EX-GWF-U1DISV                 NPF
        1         1               CHD         121         1        -1         6    1.00000000        1.00000000        1.00000000         EX-GWF-U1DISV       EX-GWF-U1DISV       EX-GWF-U1DISV            CHD-LEFT
        1         1               CHD         121         1        -1         6    1.00000000        1.00000000        1.00000000         EX-GWF-U1DISV       EX-GWF-U1DISV       EX-GWF-U1DISV           CHD-RIGHT
